pygac/gac_reader.py

```diff
@@ -460,14 +460,18 @@ def get_angles(self):
         and different ranges.
 
         Returns:
-            sat_azi: satellite azimuth angle
```

Comment:
It looks nice. Does it still follow google-style doc strings?

Reply:
Good point. No it doesn't. I've changed it according to https://stackoverflow.com/a/29343326/5703449.